data/json/items/armor.json

```diff
@@ -4238,6 +4305,28 @@
     "environmental_protection": 4,
     "flags": [ "WATERPROOF", "STURDY" ]
   },
+  {
+    "id": "helmet_acidchitin",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "biosilicified chitin helmet",
+    "description": "A helmet crafted from carefully cleaned and pruned biosilicified exoskeletons of acidic ants.  Covers the entire head; acid-resistant and very durable.",
+    "weight": 1628,
+    "volume": 11,
+    "price": 47500,
+    "to_hit": -2,
+    "bashing": 3,
+    "material": [ "acidchitin" ],
+    "symbol": "[",
+    "color": "green",
+    "covers": [ "HEAD", "EYES", "MOUTH" ],
```

Comment:
Environmental protection applies to glares and inhaled gases if it covers eyes and mouth.

Reply:
Yeah, I'm not trying to pump up resistance to those, but since environmental protection has those effects baked in and the acid resistance formula relies on environmental protection, it's difficult to balance those numbers. Still working on it.